cloud/services/compute/loadbalancers/service.go

```diff
@@ -113,6 +118,6 @@ func New(scope Scope) *Service {
 		regionalhealthchecks:    scope.Cloud().RegionHealthChecks(),
 		instancegroups:          scope.Cloud().InstanceGroups(),
 		targettcpproxies:        scope.Cloud().TargetTcpProxies(),
```

Comment:
I believe that the health checks should be created or found in the VPC Network Project. I am unsure, but the Target TCP Proxies may be too.

Reply:
@barbacbd I updated to include `regionalHealthChecks` and `targetTCPProxies`.  I also updated the tests.  I finished signing the CLA and got the confirmation emails, but this PR is still failing on that step.